app/src/main/java/com/x8bit/bitwarden/ui/tools/feature/send/SendContent.kt

```diff
@@ -48,6 +48,7 @@ fun SendContent(
         item {
             BitwardenListHeaderText(
                 label = stringResource(id = R.string.types),
+                supportingLabel = "2",
```

Comment:
Would it be overkill to make this a constant? personal aversion to "" literals. If it weren't for the click listeners we could maybe do something slick like 
```
fun LazyListScope.BitwardenGroupItemList(
    header: String,
    items: ImmutableList<BitwardenGroupItemData>,
) {
    item {
        BitwardenListHeaderText(label = header, supportingLabel = items.size.toString())
    }
    
    items(items.size) {
        BitwardenGroupItem(
            ...
        )
    }
}
``` 
where we could reuse here and the `VaultContent`

Reply:
Hmmm, we use string literals, so stuff like this in the UI right now. The Trash on the VaultScreen for instance. There is no `items` to key off of here.

```kotlin
            BitwardenListHeaderText(
                label = stringResource(id = R.string.trash),
                supportingLabel = "1",
                modifier = Modifier
                    .fillMaxWidth()
                    .padding(horizontal = 16.dp),
            )
```

Think we should look into updating those as well?

Reply:
I went ahead and updates all of them 😄